src/tests/run-study-tests/actions_on_study/study_run.py

```diff
@@ -23,9 +22,7 @@ def run(self):
         solver_full_path = str(Path(self.solver_path).resolve())
 
         command = [solver_full_path, "-i", str(self.study_path)]
```

Comment:
rename solver_full_path, i think it refer to Antares Simulator

Reply:
You're totally right.
Even in the cmake files, antares simulator executable is named **antares-solver**, which is misleading in a way